proto/buf/alpha/registry/v1alpha1/search.proto

```diff
@@ -104,6 +105,8 @@ message SearchResult {
 service SearchService {
   // Search searches the BSR.
   rpc Search(SearchRequest) returns (SearchResponse);
+  // SearchTag searches in a repository
+  rpc SearchTag(SearchTagRequest) returns (SearchTagResponse);
```

Comment:
Is this intended to search by reference, or will it, as the name suggested only search by commit tag?

A [reference](https://docs.buf.build/bsr/overview#referencing-a-module) can be a commit, tag and draft name (coming soon).